fusillade-api.yml

```diff
@@ -108,6 +108,14 @@ paths:
           in: query
           schema:
             type: string
+        - name: nonce
```

Comment:
I don't see where the `nonce` and `prompt` parameters are set in the `authorize()` method, do they need to be added?

Reply:
they are in the query_parameters in the current_request which are passed to the proxied request.